p_sets/2/encryption.rb

```diff
@@ -0,0 +1,43 @@
+module Encryption
+
+  ASCII_LETTERS = {
+    capitals: (65..90),
+    lowercase: (97..122)
+  }
+
+  def cyphertext_reporter(plaintext, integer)
+    puts format('%s', encrypt(plaintext, integer))
+  end
+
+  def encrypt(plaintext, string)
+    key = make_key(string)
+    encrypted_ascii = make_encrypted_ascii(plaintext, key)
+    make_cyphertext(encrypted_ascii)
+  end
+
+  def apply_key(character, key)
+    capital_letter?(character) ? size = :capitals : size = :lowercase
+    ascii = character.ord + key
+    ascii > ASCII_LETTERS[size].max ? ascii - 26 : ascii
```

Comment:
I feel like there should be a neater way of writing this, but I couldn't do better than 
`((key + character.ord - ASCII_LETTERS[size].min) % 26) + ASCII_LETTERS[size].min`
which isn't necessarily an improvement.